spec/features/comment_pages/comment_feed_spec.rb

```diff
@@ -134,20 +162,23 @@
         end
 
         it 'does not save unsaved changes in database' do
-          expect(model.content).not_to eq 'test comment edited'
+          expect(model.reload.content).not_to eq 'test comment edited'
         end
       end
 
       context 'when cancel is clicked' do
         it 'clears cached value' do
-          page.driver.browser.navigate.refresh
+          page.refresh
```

Comment:
Do we need to refresh at the start?

Reply:
I moved it to the `before` block. It's needed to hide the edit comment box on the page without clearing the cache.